src/_includes/partials/gam-lazy-loader.njk

```diff
@@ -7,8 +7,7 @@ document.addEventListener('DOMContentLoaded', function() {
     var isSideBarDisplayed = window.getComputedStyle(sidebar).display !== 'none';
     var topBannerAd = document.querySelector('.banner-ad-top');
     var isScreenLarge = window.innerWidth > 900;
-    var testHref = 'https://www.freecodecamp.org/news/are-you-being-micro-managed-manage-your-relationship-with-your-manager-instead-9ad10b28bcda/'
-    var topBannerTestable = window.location.href === testHref && isScreenLarge;
+    var topBannerTestable = window.location.href === isScreenLarge;
```

Comment:
Seems like this condition will always evaluate to false.

Might be better to use `isScreenLarge` in the `if` statement on line 75 to determine whether to load the banner ad or not, assuming we only want to show ads on larger screens for now.

Reply:
Also, it might be good to remove the console log near the top of this file now that we'll use GAM ads on all post pages.